products/template/build_scripts/transform.py

```diff
@@ -21,12 +46,21 @@ def intermediate():
     _run_step("historic_landmarks")
 
 
-def outputs():
+def product():
```

Comment:
from my limited research, it seems that it is possible to invoke table builds via one CLI command (`dbt run`). I wonder if there is a similar python way to do that...

Reply:
I think it'd be `dbt_runner.invoke(["run"])` and with the wrapper I wrote above it'd be `_run_dbt_step(["run"])`

Reply:
I thought you were using `dbt` to build the tables so was confused having multiple steps in the `product` fn. Makes sense